homeassistant/const.py

```diff
@@ -480,8 +480,10 @@ class Platform(StrEnum):
 STATE_ALARM_DISARMING: Final = "disarming"
 STATE_ALARM_TRIGGERED: Final = "triggered"
 STATE_LOCKED: Final = "locked"
+STATE_UNLATCHED: Final = "unlatched"
```

Comment:
Please don't change the common constants. If you need a constant define it in the August integration.